lib/vmdb/plugins.rb

```diff
@@ -4,29 +4,53 @@ class Plugins
 
     attr_reader :vmdb_plugins
     attr_reader :registered_automate_domains
+    attr_reader :registered_provider_plugins
 
     def initialize
       @registered_automate_domains = []
+      @registered_provider_plugins = []
       @vmdb_plugins = []
+
+      Rails.application.railties.each do |railtie|
+        next unless railtie.class.name.start_with?("ManageIQ::Providers::") || railtie.try(:vmdb_plugin?)
```

Comment:
@durandom Would you mind adding this to the engine.rb on the plugin generator?  It would simplify this code.
```ruby
def vmdb_plugin?
  true
end
```